services/QuillLMS/client/app/bundles/Staff/components/comprehension/changeLog/changeLog.tsx

```diff
@@ -0,0 +1,253 @@
+import * as React from "react";
+import * as _ from 'underscore';
+import { RouteComponentProps } from 'react-router-dom'
+import { useQuery } from 'react-query';
+import { firstBy } from "thenby";
+import ReactTable from 'react-table';
+import qs from 'qs';
+import * as _ from 'lodash'
+import DateTimePicker from 'react-datetime-picker';
+
+import { renderHeader } from '../../../helpers/comprehension';
+import { sort } from '../../../../../modules/sortingMethods.js';
+import { fetchChangeLog, fetchActivity } from '../../../utils/comprehension/activityAPIs';
+import { DropdownInput, Spinner, } from '../../../../Shared/index';
+
+interface ChangeLogProps {
+
+}
+
+const ChangeLog = ({ history, match }) => {
+  const { params } = match;
+  const { activityId, } = params;
+  const SESSION_INDEX=1;
+  const initialStartDateString = window.sessionStorage.getItem(`${SESSION_INDEX}startDate`) || '';
+  const initialEndDateString = window.sessionStorage.getItem(`${SESSION_INDEX}endDate`) || '';
```

Comment:
I'm not really sure why this is called `SESSION_INDEX` because I don't see the relation to the activity sessions index? I think we want to be making a `CHANGE_LOG` const stored in that comprehension constants file and then use it here for these session storage keys